src/modules/drawing/components/DrawingToolboxButton.vue

```diff
@@ -27,7 +33,10 @@ export default {
     },
     emits: ['setDrawingMode'],
     computed: {
-        buttonIcon: function () {
+        ...mapState({
+            uiMode: (state) => state.ui.mode,
+        }),
```

Comment:
Could you move this Vuex binding to the parent component (the one that is already bound with it) and pass that as a prop?
We try to keep at a minimum the amount of component that are linked to the store, typically only binding the store on the top component of each modules.

Reply:
Ok 😃